_scripts/reconfigure_oauth2

```diff
@@ -12,9 +12,13 @@ set -e
 ENV_FILE=${1}
 shift
 
+## env var for oauth2
 VAR=${1}
 shift
 
+## env var for authorized emails
+VAR2=${2}   
```

Comment:
Something's wrong with the args here. You use `shift` (which pops the first arg off and the second arg becomes $1 for each time you call `shift`), so it shouldn't be $2 here. When I changed it to $1 and I inspected the value it was `VAR=default=yes` which also seems wrong.

Reply:
These should not be called `VAR` and `VAR2` please pick descriptive names.

Reply:
$2 after the shift was correct because $1 is used elsewhere, but I'll rename the vars

Reply:
fixed in upcoming commit